src/OpenLoco/src/GameSaveCompare.h

```diff
@@ -1,5 +1,5 @@
-#pragma once
-
+#pragma once

```

Comment:
What's different here? As long as we didn't _add_ a byte order marker (BOM) it's fine with me 😄

Reply:
line endings were not normalised :(

master:
![image](https://github.com/OpenLoco/OpenLoco/assets/7483209/1576ec74-c0c2-4a66-be0a-bec8f765d4df)

this pr:
![image](https://github.com/OpenLoco/OpenLoco/assets/7483209/f70b5686-c7dd-47e5-8236-01b7780415b0)